src/test/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfigTest.java

```diff
@@ -68,6 +71,25 @@ public void testUndefinedURL() throws Exception {
   }
 
   @Test
+  public void testAvroCompressionSettings() {
+    for (String codec : HdfsSinkConnectorConfig.AVRO_SUPPORTED_CODECS) {
+      Map<String, String> props = new HashMap<>(this.properties);
+      props.put(HdfsSinkConnectorConfig.AVRO_CODEC_CONFIG, codec);
+      HdfsSinkConnectorConfig config = new HdfsSinkConnectorConfig(props);
+      Assert.assertNotNull(config.getAvroCodec());
+    }
+  }
+
+  @Test(expected = ConfigException.class)
+  public void testUnsupportedAvroCompressionSettings() {
+    // test for an unsupported codec.
+    this.properties.put(HdfsSinkConnectorConfig.AVRO_CODEC_CONFIG, "abc");
+
+    new HdfsSinkConnectorConfig(properties);
+    Assert.assertTrue("Expected the constructor to throw an exception",false);
```

Comment:
Formatting will show that there's a missing whitespace character after comma in `exception",false);`

Reply:
Thanks, @kkonstantine. Added the whitespace.